overlord/servicestate/servicestate.go

```diff
@@ -39,7 +40,7 @@ type Instruction struct {
 
 type ServiceActionConflictError struct{ error }
 
-func Control(st *state.State, appInfos []*snap.AppInfo, inst *Instruction) (*state.TaskSet, error) {
+func Control(st *state.State, appInfos []*snap.AppInfo, inst *Instruction, context *hookstate.Context) (*state.TaskSet, error) {
```

Comment:
this doesn't have a doc comment, it's probably merits one so that we can also explain context